src/Generators/Ics.php

```diff
@@ -37,7 +37,7 @@ public function generate(Link $link): string
 
         if ($link->allDay) {
             $url[] = 'DTSTART:'.$link->from->format($dateTimeFormat);
-            $url[] = 'DURATION:P1D';
+            $url[] = 'DURATION:P'.($link->from->diff($link->to)->days).'D';
```

Comment:
I think it should be at least 1, `P0D` looks weird, probably not what a developer expected.

```php
(new DateTime())->diff(new DateTime('+23h'))->days
```
returns `0`.

_______
How about `max(1, $link->from->diff($link->to)->days)` instead?

Reply:
I think that's a good idea, added your suggestion.